src/content-helper/editor-sidebar/smart-linking/component.tsx

```diff
@@ -676,7 +674,6 @@ export const SmartLinkingPanel = ( {
 				{ error && (
 					<Notice
 						status="info"
-						isDismissible={ true }
 						onRemove={ () => setError( null ) }
```

Comment:
Since we no longer can dismiss this notice, we can also remove the onRemove attribute, as it will not be used. 
```suggestion
```